src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -60,6 +60,8 @@
 
     private static int newOrderTxnId = -1;
     private static int numWarehouses = 10;
+    private static int startWarehouseId = -1;
+    private static int totalWarehousesAcrossExecutions = 10;
```

Comment:
Rename these to `startWarehouseIdForShard` and `totalWarehousesAcrossShards`

Reply:
Done.